test/ncentSDK.test.js

```diff
@@ -0,0 +1,75 @@
+global.Buffer.prototype.__proto__ = global.Uint8Array.prototype;
```

Comment:
could you structure the tests like:
test/unit/*
test/feature/*

and sub folders based on wheere the file you are testing is from -- ex: controller/ model/ service/ (if neeeded)

Reply:
could u also use 'context', 'describe' and 'it' blocks